protos/proto/aptos/indexer/v1/filter.proto

```diff
@@ -0,0 +1,63 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+syntax = "proto3";
+
+package aptos.indexer.v1;
+
+import "aptos/transaction/v1/transaction.proto";
+
+message LogicalAndFilters {
+  repeated BooleanTransactionFilter filters = 1;
+}
+
+message LogicalOrFilters {
+  repeated BooleanTransactionFilter filters = 1;
+}
+
+message TransactionRootFilter {
+  optional bool success = 1;
+  optional aptos.transaction.v1.Transaction.TransactionType transaction_type = 2;
+}
+
+message EntryFunctionFilter {
+  optional string address = 1;
+  optional string module_name  = 2;
+  optional string function = 3;
+}
+
+message UserTransactionPayloadFilter {
+  optional EntryFunctionFilter entry_function_filter = 1;
+}
+
+message UserTransactionFilter {
+  optional string sender = 1;
+  optional UserTransactionPayloadFilter payload_filter = 2;
+}
+
+
+message MoveStructTagFilter {
```

Comment:
Just checking, do we actually support filtering just on module name (for example) without setting the other fields?

Reply:
We do.